lib/licensee/matchers/gemspec.rb

```diff
@@ -1,18 +1,36 @@
 module Licensee
   module Matchers
     class Gemspec < Licensee::Matchers::Package
+      # a value is a string surrounded by any amount of whitespace
+      # optionally ended with (non-captured) ".freeze"
+      VALUE_REGEX = /\s*[\'\"]([a-z\-0-9\.]+)[\'\"](?:\.freeze)?\s*/i
+
+      # an array contains one or more values. all values, or array itself,
+      # can be surrounded by any amount of whitespace.  do not capture
+      # non-value groups
+      ARRAY_REGEX = /\s*\[#{VALUE_REGEX}(?:,#{VALUE_REGEX})*\]\s*/i
+
       DECLARATION_REGEX = /
-        ^\s*[a-z0-9_]+\.([a-z0-9_]+)\s*\=\s*[\'\"]([a-z\-0-9\.]+)[\'\"]\s*$
-        /ix
+        ^\s*[a-z0-9_]+\.([a-z0-9_]+)\s*\=#{VALUE_REGEX}$
+      /ix
 
       LICENSE_REGEX = /
-           ^\s*[a-z0-9_]+\.license\s*\=\s*[\'\"]([a-z\-0-9\.]+)[\'\"]\s*$
-           /ix
+        ^\s*[a-z0-9_]+\.license\s*\=#{VALUE_REGEX}$
+      /ix
+
+      LICENSE_ARRAY_REGEX = /
+        ^\s*[a-z0-9_]+\.licenses\s*\=#{ARRAY_REGEX}$
+      /ix
 
       private
 
       def license_property
         match = @file.content.match LICENSE_REGEX
+        return match[1].downcase if match && match[1]
+
+        # this will capture each value in the license array
+        # for now only return the first match
+        match = @file.content.match LICENSE_ARRAY_REGEX
```

Comment:
Ideally, we'd abstract things such that a project file could have multiple licenses, but for now, I agree with @mlinksva that "other" would be the consistent response here (but I'd leave the other logic intact to allow for that). Perhaps expose a `matches` method for now (in addition to `match`)?

Reply:
I avoided adding any new fields at a broad scope because I wasn't sure how they would be used.  It definitely seems worth thinking about for the future.